data-processing-service/src/main/java/gov/cdc/dataprocessing/utilities/component/organization/OrganizationRepositoryUtil.java

```diff
@@ -105,20 +107,15 @@ public Organization findOrganizationByUid(Long orgUid) {
     public long createOrganization(OrganizationVO organizationVO)
             throws DataProcessingException {
         Long organizationUid = 121212L;
+        long oldOrgUid = organizationVO.getTheOrganizationDT().getOrganizationUid();
```

Comment:
needed this

Reply:
follow the flow, you will know why, we need orgUid on organization container to be negative in the upper level. The setFalseToNew method use this negative id to set orgUid to associate object (Collection of Act Relationship, Collection of Role, Collection of Participation).
This method also return the new orgUid at the end